<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseManage</class>
 <widget class="QWidget" name="WarehouseManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>20</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolBox" name="warehousetoolBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>131</width>
     <height>431</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Mainpage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>379</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Main</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>112</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="MIpushButton">
        <property name="text">
         <string>Material Inquiry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MMpushButton">
        <property name="text">
         <string>Material Modify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MSpushButton">
        <property name="text">
         <string>Material Storage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MOpushButton">
        <property name="text">
         <string>Material Outbound</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MDpushButton">
        <property name="text">
         <string>Material Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IOpushButton">
        <property name="text">
         <string>Inquiry Outbound</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Otherpage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>379</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Other</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>63</y>
       <width>111</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="CPpushButton">
        <property name="text">
         <string>Change Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SOpushButton">
        <property name="text">
         <string>Sign Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ESpushButton">
        <property name="text">
         <string>Exit System</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="warehousestack">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>9</y>
     <width>611</width>
     <height>411</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>486</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="managemlabel">
        <property name="styleSheet">
         <string notr="true">color:black;

font: 18pt &quot;华文行楷&quot;;
</string>
        </property>
        <property name="text">
         <string>Warehouse:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="usernamelabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;华文行楷&quot;;
color: rgb(0, 0, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="timelabel">
        <property name="styleSheet">
         <string notr="true">color:black;

font: 18pt &quot;华文行楷&quot;;</string>
        </property>
        <property name="text">
         <string>SystemTime:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="systemtimelabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;华文行楷&quot;;
color: rgb(0, 0, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="testpushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IE云智能科技（杭州）有限公司</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
